src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:string';
+@use 'sass:math';
+
+$button-color: #00acdc;
+$main-text-color: #060b35;
+$second-text-color: #616070;
+$price-in-grn: 69999;
+
+@function converter($sum) {
+  $factor: 1000;
+
+  @return math.div(round(($sum / 31832.19) * $factor), $factor);
```

Comment:
The `math.div` function is used here, but ensure that the `round` function is correctly applied. If `round` is not a valid function or not imported, it will cause an error. Consider using `math.round` if available.